opencti-platform/opencti-front/src/private/components/entities/events/EventEditionOverview.js

```diff
@@ -134,17 +135,12 @@ const EventEditionOverviewComponent = (props) => {
       if (name === 'x_opencti_workflow_id') {
         finalValue = value.value;
       }
-      eventValidator
-        .validateAt(name, { [name]: value })
-        .then(() => {
-          editor.fieldPatch({
-            variables: {
-              id: event.id,
-              input: { key: name, value: finalValue ?? '' },
-            },
-          });
-        })
-        .catch(() => false);
+      editor.fieldPatch({
```

Comment:
It's not a good solution to remove the validation here because all the others property will not be validated.
We need to re-work the stop_time validator.

Reply:
I find that
`    stop_time: Yup.date()
      .when('start_time', (startTime, field) => {
        return startTime ? field.min(Yup.ref('start_time'), "The end date can't be before start date") : field;
      })
      .typeError(t('The value must be a datetime (yyyy-MM-dd hh:mm (a|p)m)'))
      .nullable(),`
      
Seems to working as expected

Reply:
If it's ok for you, we need to propagate this changes on MalwareEditionDetails

Reply:
the properties seems validated anyway because the validator was present in the Formik...
but i'm going to try your solution and keep it if it works :)

Reply:
--> fist solution kept because not redondant